ppmpy/ppm.py

```diff
@@ -10130,12 +10178,115 @@ def __init__(self, file_path, verbose=3, geometry='spherical'):
             self.__messenger.warning(wrng)
             self.__geometry = 'spherical'
 
-        if self.__read_rprof_file(file_path) != 0:
-            return
+        if self.__bqav:
+            if self.__read_rprofbqav_file(file_path) != 0:
+                return
+        else:
+            if self.__read_rprof_file(file_path) != 0:
+                return
 
         self.__is_valid = True
 
+
+    def __read_rprofbqav_file(self, file_path):
+        '''
+        Reads a single frombqav .rprof file written by PPMstar 2.0.
+
+        Parameters
+        ----------
+        file_path: string
+            Path to the .rprof file.
+
+        Returns
+        -------
+        int
+            0 on success.
+        NoneType
+            Something failed.
         
+        Variables
+        ---------
+        los: dictionary
```

Comment:
no los variable here, maybe just delete variable section?

Reply:
i will test this now

Reply:
This is now fixed. Please try again.